contrib/python/pypodman/pypodman/lib/actions/_create_args.py

```diff
@@ -0,0 +1,394 @@
+            help=('Limit read rate (IO per second) from a device'
+                  ' (e.g. --device-read-iops=/dev/sda:1000)'
+                  'The option can be set multiple times.'),
+        )
+        parser.add_argument(
+            '--device-write-bps',
+            action='append',
+            metavar='LIMIT',
+            help=('Limit write rate (bytes per second) to a device'
+                  ' (e.g. --device-write-bps=/dev/sda:1mb)'
+                  'The option can be set multiple times.'),
+        )
+        parser.add_argument(
+            '--device-write-iops',
+            action='append',
+            metavar='LIMIT',
+            help=('Limit write rate (IO per second) to a device'
+                  ' (e.g. --device-write-iops=/dev/sda:1000)'
+                  'The option can be set multiple times.'),
+        )
+        parser.add_argument(
+            '--dns',
+            action='append',
+            metavar='SERVER',
+            help=('Set custom DNS servers.'
+                  'The option can be set multiple times.'),
+        )
+        parser.add_argument(
+            '--dns-option',
+            action='append',
+            metavar='OPT',
+            help=('Set custom DNS options.'
+                  'The option can be set multiple times.'),
+        )
+        parser.add_argument(
+            '--dns-search',
+            action='append',
+            metavar='DOMAIN',
+            help=('Set custom DNS search domains.'
+                  'The option can be set multiple times.'),
+        )
+        parser.add_argument(
+            '--entrypoint',
+            help=('Overwrite the default ENTRYPOINT of the image.'),
+        )
+        parser.add_argument(
+            '--env',
+            '-e',
+            action='append',
+            help=('Set environment variables.'),
+        )
+        parser.add_argument(
+            '--env-file',
+            help=('Read in a line delimited file of environment variables,'
+                  ' on the remote host.'),
+        )
+        parser.add_argument(
+            '--expose',
+            metavar='RANGE',
+            help=('Expose a port, or a range of ports'
+                  ' (e.g. --expose=3300-3310) to set up port redirection.'),
+        )
+        parser.add_argument(
+            '--gidmap',
+            metavar='MAP',
+            help=('GID map for the user namespace'),
+        )
+        parser.add_argument(
+            '--group-add',
+            action='append',
+            metavar='GROUP',
+            help=('Add additional groups to run as'))
+        parser.add_argument('--hostname', help='Container host name')
+
+        volume_group = parser.add_mutually_exclusive_group()
+        volume_group.add_argument(
+            '--image-volume',
+            choices=['bind', 'tmpfs', 'ignore'],
+            metavar='MODE',
+            help='Tells podman how to handle the builtin image volumes')
+        volume_group.add_argument(
```

Comment:
duplicated call

Reply:
Updated code so more explicit about the add_mutually_exclusive_group() happening here.